conf/shiro.ini.template

```diff
@@ -110,6 +110,7 @@ admin = *
 # uncomment the line second last line (/** = anon) and comment the last line (/** = authc)
 #
 /api/version = anon
+/api/cluster/address = anon
```

Comment:
Should this be only accessed by admin ?

Reply:
Because `navbar` is the page that will be called when the zeppelin home page is initialized.
So authorize everyone, all can access `/api/cluster/nodes` this REST interface.
Otherwise, zeppelin will automatically pop up the `LOGIN page` because it accesses the page controlled by the permission.

Reply:
In TODOs, there's one task about displaying zeppelin service and interpreter log ? Does it only apply for cluster mode, because I think it would be good to have this feature for all modes.

Reply:
In addition, in the cluster mode, cluster metadata is also required to manage session information in all zeppelin servers, and the session is managed uniformly.

The interpreter process's service discovery through metadata is also more important than the current thrift callback. It can be found in any zeppelin server or interpreter process through metadata service discovery.

So I suggest that metadata management be started on either stand-alone or cluster.
Then the instant is a single machine, there will also be this Cluster management page. But that is a long time later.